features/payment-flows-and-management/payment-links.md

```diff
@@ -94,6 +94,123 @@ You can also customize a specific payment link by including the payment\_link\_c
+
+Step 2: Locate the page to manage the DNS for your domain
+
+> Now that you've successfully logged in, locate the section within your provider's control panel where you can manage the DNS records for your domain.
+
+Step 3: Create CNAME record
+
+> In your DNS control panel, create a new record that associates your chosen subdomain with 'hyperswitch payment link'. Your DNS provider will typically prompt you to specify the record type, name, value, and TTL (Time To Live) or expiration when adding a new record.
+
+Enter the following values and save the new DNS record.
+
+>| FIELD | INSTRUCTIONS | DESCRIPTION |
+|----------|----------|----------|
+| Type | Select `CNAME` from the dropdown | What kind of DNS record this is. |
+| Name | if your custom domain is `pay.xyz.com`, enter hyperswitch-paymentlink| For CNAME records, this field is the first part of your subdomain (the part leading up to the first period).|
+| Value |Enter `sandbox.hyperswitch.juspay.in` | This is what the new subdomain record points to–in this case, Hyperswitch .Some providers may expect a trailing period (.) after the CNAME value. Make sure to verify that your CNAME value matches the format your provider expects. |
+| TTL/Expiry | Enter `300` | An expiration of 5 minutes (300 seconds) is OK. Your DNS provider might not allow you to change the TTL value. If this field is missing or you can’t change it, it’s safe to ignore this part of the configuration. |
+
+
+Step 4: Create your TXT record
+
+>Navigate to your DNS control panel and proceed to add a new TXT record.
+
+>> This TXT record is essential for domain ownership verification. It's a necessary step to obtain TLS certificates for your domain, ensuring secure payment processing.
+
+>Enter these values and save the new DNS record:
+
+>| FIELD | INSTRUCTIONS | DESCRIPTION |
+|----------|----------|----------|
+| Type | Select `TXT` from the dropdown | What kind of DNS record this is. |
+| Name | If your custom domain is `pay.xyz.com`, enter _acme-challenge.hyperswitch-paymentlink| For TXT records, this field is the subdomain portion of your domain. |
+| Value | Copy the TXT value that is given by us and paste | This is a long, unique string used for domain verification |
+| TTL/Expiry | Enter `300` | An expiration of 5 minutes (300 seconds) is OK. Your DNS provider might not allow you to change the TTL value. If this field is missing or you can’t change it, it’s safe to ignore this part of the configuration.|
+
+
+Step 5. Verify your CNAME record setup
+
+> After you save your DNS record, verify that it has the correct values.
+
+>> Please allow up to 10 minutes for your DNS provider to update its nameservers.
+>> Replace "pay.xyz.com" with your custom domain in the command below, and then run it in your terminal:
+```shell
+$ nslookup -querytype=CNAME pay.xyz.com
+```
+
+your should get a output like this
+
+```shell
+<your subdomain> 	canonical name = sandbox.hyperswitch.io.
```

Comment:
Can we also check this. We should mention the what should they expect in production and sandbox both